### PR TITLE
Update HL tree to support Hats App v2.4 UI improvements (authority and responsibility cards)

### PR DESCRIPTION
**Pull request by nintynick**
Testing if github is a good way to do this to review the diffs in PRs. 

In the governance facilitator persona, I made a number of changes to the tree (process described below). I am sharing this series of changes for review by the rest of the org. 

Process:
- I took every authority or responsibility that was too long as a title and pasted it into description, then wrote a shorter title
- added some information in descriptions where necessary
- added a few links where appropriate
- added images for telegram, github, charmverse, daohaus, discourse forum, zoom where appropriate

**Goal: Can you review the following _A. hats_, using the following _B. views_, for the review _C. criteria_ below?**

A. Hats for Review:
- 1.2 Hats protoDAO
- 1.2.1 Hats Community Admin
- 1.2.1.1 Hats Community Member
- 1.2.1.2 Hats Active Contributor
- 1.2.1.3 Hats Steward
- 1.2.4.1 Hats Community Coordinator
- 1.2.4.2 Hats Arbitrator
- 1.2.4.3 protoDAO Coordinator
- 1.2.4.4 Modules Registry Curator
- 1.2.4.5 Module Curation Manager

B. Review Views:
1. Review PR changes on the JSON file here in Github
2. Review draft changes by importing the latest JSON file (on this branch) into the Hats App
3. Review a "preview version" of the deployed changes (on testnet) https://bucket.hatsprotocol.xyz/trees/5/415?flipped=true

C. Review Criteria
- new titles are accurate representations of their descriptions
- content is in the correct places, images are correct
- proofreading

Tx call data:
0xac9650d80000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000000000000000000000000000000140000000000000000000000000000000000000000000000000000000000000024000000000000000000000000000000000000000000000000000000000000003400000000000000000000000000000000000000000000000000000000000000440000000000000000000000000000000000000000000000000000000000000054000000000000000000000000000000000000000000000000000000000000006400000000000000000000000000000000000000000000000000000000000000740000000000000000000000000000000000000000000000000000000000000084000000000000000000000000000000000000000000000000000000000000009400000000000000000000000000000000000000000000000000000000000000a4000000000000000000000000000000000000000000000000000000000000000c4f737401e000000010002000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000042697066733a2f2f6261666b726569616464746836766c33376a7572343677336e64346971656c37373569637a6b7267686c76723571613572753267333375357779340000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c4f737401e000000010002000100010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000042697066733a2f2f6261666b726569627165756f3362746962696f6d7a76676f33737a327477356a74697a61657a3565686a6a6171327267786179736b6a71657168710000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c4f737401e000000010002000100020000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000042697066733a2f2f6261666b72656963627462636161326c71716634736e6267366a7268676874377436636532796a7366703374726475353337756a79796b37666f790000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c4f737401e000000010002000100030000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000042697066733a2f2f6261666b726569687233646965613232726e786c74653669626d6b786a6c736575347861786f623676376b637678706275767776666b68687769340000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c4f737401e000000010002000400020000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000042697066733a2f2f6261666b7265696234677a366e326264346d6473747a79646f62356e7a336e746a34616f666569777477716235636661697332686c7973613677710000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c4f737401e000000010002000400030000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000042697066733a2f2f6261666b726569653336746578626164736779776736636f7874357a37656a3667636b6b6c696274763569727234626734626634347070716878690000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c4f737401e000000010002000400040000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000042697066733a2f2f6261666b72656967637a766b75616876776677633570686f75636834786a6a7a7a626f73367a653278627965347462616535753535716669366f790000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c4f737401e000000010002000400050000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000042697066733a2f2f6261666b726569637432796a62787233787a706472367a6f79766d67626e7435327675617766726d7236757966666a6264667a37346235786b35790000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c4f737401e000000010002000100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000042697066733a2f2f6261666b72656965683733776a767074356b677179653769793361766466373733327a6d36336f61746264636e6e6e7170716836616e6e757462690000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c4f737401e000000010002000400010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000042697066733a2f2f6261666b72656965686c6d613278356b67346c75616a62716c78356e6f653677787969746437727171706c726d797570706934747a65336e62793400000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000

### CODE REVIEW

File: chain-10-tree-1.json [hunk 1]
```diff
@@ -93,11 +93,13 @@
...
             "link": "",
-            "label": "Develop a strong contributor community infrastructure around Hats Protocol"
+            "label": "Develop community infrastructure",
+            "description": "Develop a strong contributor community infrastructure around Hats Protocol"
           },
           {
             "link": "",
-            "label": "Develop sticky network effects associated with the canonical version of the protocol"
+            "label": "Develop network effects",
+            "description": "Develop sticky network effects associated with the canonical version of the protocol"
```
Comment (nintynick) [inline chain-10-tree-1.json:102]:
```suggestion
            "description": "TESTDevelop sticky network effects associated with the canonical version of the protocol"
```

File: chain-10-tree-1.json [hunk 2]
```diff
@@ -324,26 +343,39 @@
...
+            "label": "Follow Community Agreements",
+            "description": "Conduct yourself in a manner that is consistent with the Hats Community Agreements and Code of Conduct, which you signed in order to claim this hat"
           }
         ],
         "authorities": [
           {
             "link": "https://telegram.me/collablandbot?start=VFBDI1RFTCNDT01NIy0xMDAxODUxMjg4MjQy",
-            "label": "Telegram: Read & write access in the Hats Community Telegram group"
+            "label": "Access to the Hats Community Telegram group",
+            "description": "Read & write access in the Hats Community, Season 1 Telegram group",
```
Comment (spengrah) [inline chain-10-tree-1.json]: We will need to update this for Season 2

File: chain-10-tree-1.json [hunk 3]
```diff
@@ -324,26 +343,39 @@
...
+            "gate": "",
+            "imageUrl": "ipfs://bafkreig4c6wczcswoxhsvejlk47mnyfcwi6jch3sknfh6cwtootz7vag6a"
```
Comment (spengrah) [inline chain-10-tree-1.json:256]: meta: images are difficult to review in the text diff method

File: chain-10-tree-1.json [hunk 4]
```diff
@@ -393,37 +425,50 @@
...
           {
             "link": "https://app.charmverse.io/hats-protocol/page-0225506388762724",
-            "label": "Record contributions as appropriate using the Hats contribution tracking system"
+            "label": "Record contributions",
+            "description": "Record contributions as appropriate using the Hats contribution tracking system"
           }
         ],
         "authorities": [
           {
             "link": "",
```
Comment (spengrah) [inline chain-10-tree-1.json:348]: link to the [contribution instructions](https://app.charmverse.io/hats-protocol/how-to-submit-a-contribution-for-protorep-consideration-23301472932733813)

File: chain-10-tree-1.json [hunk 5]
```diff
@@ -491,37 +536,51 @@
         "responsibilities": [
           {
             "link": "",
-            "label": "Declare your intentions for how you intend to contribute to Hats Protocol and/or the Hats protoDAO for the coming Season"
+            "label": "Declare seasonal contribution intentions",
```
Comment (spengrah) [inline chain-10-tree-1.json:419]: "intentions for how you intend" is awkward

Suggestion:
"Declare your intentions for how you will contribute to Hats Protocol and/or the Hats protoDAO for the coming Season"

File: chain-10-tree-1.json [hunk 6]
```diff
@@ -626,25 +690,26 @@
...
-          {
-            "link": "",
-            "label": "Tracking and assessing the contributions of community members"
+            "label": "Track and assess contributions",
+            "description": "Tracking and assessing the contributions of community members"
           },
           {
             "link": "",
-            "label": "Recommending and proposing the granting of Active Contributor hats to those who are assessed as making active, meaningful contributions to Hats, as well as the revocation of Active Contributor hats for those who have not been actively contributing over a reasonable time period"
+            "label": "Proposing Active Contributor hat wearers",
```
Comment (spengrah) [inline chain-10-tree-1.json]: "Propose Contributor hat wearers"

**Review by spengrah — APPROVED** [review]
**Content**: Overall, this look great! I made a handful of suggestions for small text changes, but I'm good with it as is / trust you to make the necessary changes so I'm approving the PR.

**Meta (format, process, etc)**: Pretty good! At least, its pretty good for somebody who is used to github / git formatting. 
- Reviewing image changes requires opening a new tab and loading the IPFS asset, which is not difficult but takes you out of the review flow
- Changes are clearly marked
- Helpful to expand the entire document to show the context of each hat (compared to only viewing the changes themselves)
- Easy to comment on changes and make suggestions

**Comment by nintynick** [comment]
integrated all changes from @spengrah's review, will merge after successful deployment to OP

**Comment by spengrah** [comment]
latest changes look good!

**Comment by nintynick** [comment]
@scottrepreneur paired on this with spencer, we got an AlreadyWearingHat() error when attempting to execute calldata after uploading the JSON file at this commit (https://github.com/Hats-Protocol/chain-10-tree-1/pull/1/commits/7bffea183dba99ba4b7932b351ff29db4ebd12e6). here's the tenderly simulation from safe for this error: https://dashboard.tenderly.co/public/safe/safe-apps/simulator/07f7f892-4426-48a9-83a8-3f4acea8a713

in debugging, we noticed that there were 180 current wearers of the community member hat, versus 170 in the JSON file in this repo. we were trying to think through what is happening... it seems like the app is getting tripped up on a diff between wearers, and trying to mint new hats in an unexpected way based on the diff.

we were able to workaround this error by manually removing all of the wearer addresses from the JSON here from the Community Member hat, as well as the list of wearers in the Steward hat in the two most recent commits (for some reason). Spencer was able to get a successful simulation by only removing the Community Member hat wearers from the JSON.

upon simulating the transaction in safe using the new JSON (now the current commit in this branch), we are getting a success https://dashboard.tenderly.co/public/safe/safe-apps/simulator/afb79739-83cd-4e29-9f03-c8746dbad4f5

will test deploying from Safe / Top Hat now.

**Comment by nintynick** [comment]
brainstorming possible cases we probably want to test for:
- new hat claims that are not reflected in JSON that is being deliberated on (these hats should stay in the tree)
- new renouncements or revocations that are not reflected in JSON that is being deliberated on (these hats should not be re-minted - may be a challenging case to solve for)

**Comment by spengrah** [comment]
These changes have now been executed onchain, and I'm noticing that the authority images are not being displayed. For example, below is a screenshot of the Module Curation Manager authorities, to which [images were meant to be added](https://github.com/Hats-Protocol/chain-10-tree-1/blob/21ca6b191fdcf5418e0c7ab5172130838ab690c9/chain-10-tree-1.json#L836).

image

I do see the image urls when doing a new JSON export from the app; they're just not showing up in the UI.

**Pull request merged**